docs/HowToFAQ/FAQFunctionUnroll.md

```diff
@@ -0,0 +1,36 @@
+---
+title: A function seems to work just once. How do I get it to apply a bunch of times?
+---
+
+## Question
+
+A function seems to work just once. How do I get it to apply a bunch of times?
+Here is an example:
+```
+{% include_relative FAQFunctionUnroll0.dfy %}
+```
+
+The latter two lemmas will not prove without uncommenting the application of the lemma for one less iteration.
+How can I get this to prove automatically?
+
+## Answer
+
+Function bodies are transparent. That is, when there is a call to a function in Dafny code, Dafny will insert inline
```

Comment:
This is not strictly true; not sure whether it matters in the context of this explanation

Reply:
Reworded. Tell me if this is better or not.

Reply:
(Side note: It's harder to see the follow-up if you mark the comment resolved)